components/photoslider.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Box, Button, Container, HStack } from '@chakra-ui/react';
+
```

Comment:
I've also added some sample data in `/img/exampleClubInfo.json`, so import that json file and pull all the data from there instead of hard coding it; that'll make life a lot easier down the line when migrating to MongoDB pulling data